packages/console/src/pages/EnterpriseSsoDetails/Connection/index.tsx

```
@@ -55,17 +62,72 @@ function Connection<T extends SsoProviderName>({ isDeleted, data, onUpdated }: P
         return;
       }
 
-      const updatedSsoConnector = await api
-        // TODO: @darcyYe add console test case to remove attribute mapping config.
-        .patch(`api/sso-connectors/${ssoConnectorId}`, {
-          json: { config: cleanDeep(formData) },
-        })
-        .json<SsoConnectorWithProviderConfigWithGeneric<T>>();
+      try {
+        const updatedSsoConnector = await api
+          // TODO: @darcyYe add console test case to remove attribute mapping config.
+          .patch(`api/sso-connectors/${ssoConnectorId}`, {
+            json: { config: cleanDeep(formData) },
+          })
+          .json<SsoConnectorWithProviderConfigWithGeneric<T>>();
+
+        toast.success(t('general.saved'));
+        onUpdated(updatedSsoConnector);
+
+        reset(updatedSsoConnector.config);
+      } catch (error: unknown) {
+        if (error instanceof HTTPError) {
```

Should throw the original error on "else"